sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputForbidCertainMethodCheck.java

```diff
@@ -4,6 +4,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 package com.github.sevntu.checkstyle.checks.coding;
 
+import java.util.function.Supplier;
```

Comment:
this line caused a big changes in UTs expected results, please change example to avoid extra import or remove empty line between imports to keep code on the same line further down.